src/ServiceControl/SagaAudit/AuditRetentionCustomCheck.cs

```diff
@@ -0,0 +1,45 @@
+﻿namespace ServiceControl.SagaAudit
+{
+    using System;
+    using System.Threading.Tasks;
+    using NServiceBus.CustomChecks;
+    using Raven.Client;
+    using ServiceBus.Management.Infrastructure.Settings;
+
+    class AuditRetentionCustomCheck : CustomCheck
+
+    {
+        readonly IDocumentStore _documentStore;
+        readonly bool _auditRetentionPeriodIsSet;
+
+        public AuditRetentionCustomCheck(IDocumentStore documentStore, Settings settings, TimeSpan? repeatAfter = null)
+            : base("Saga audit data retention check", "ServiceControl Health", repeatAfter.HasValue ? repeatAfter : TimeSpan.FromHours(24))
```

Comment:
I think the longest duration we have on another check is 1 hour. It means that a new instance may take a day to report the problem if the startup invocation happens before the first saga audit message is ingested.

I don't necessarily think it's an issue to only run every 24 hours, but I would consider dropping it down. Some points to consider:
- This check is not particularly taxing. It's a query on an existing index
- Custom Check events are only fired on state changes. Even if this fails once per hour forever, from an external perspective only a single event will be emitted.